rpc/walletrpc/methods_test.go

```diff
@@ -93,6 +76,7 @@ func test_wallet_getbalance(w *wallet.Wallet) (*wallet.Balance, error){
 		return nil,err
 	}
 	r:=b.(*wallet.Balance)
+	fmt.Printf("test_wallet_getbalance  UnspendAmount:%v\n",r.UnspendAmount)
```

Comment:
1.) test-case should use `assert` to test if the actual result is expected. 
2.) if you want to test the output of a program, please use https://golang.org/pkg/testing/#hdr-Examples